.devcontainer/setup.py

```diff
@@ -89,7 +89,7 @@ def setup():
         # Having fsync off in dev environment is fine.
         "sed -i -e \"s/#fsync = on/fsync = off/\" /srv/postgres14/postgresql.conf",
         "echo 'host    all         all     0.0.0.0/0     trust' >> /srv/postgres14/pg_hba.conf",
-        "supervisorctl restart postgresql",
+        # "supervisorctl restart postgresql",
```

Comment:
This still fails, but we don't necessarily have to restart postgresql during build time - it will start on container launch.